### DISCUSSION TITLE
专业版无故跳成免费版

### DISCUSSION THREAD

**Discussion by hzch170** [getlantern/lantern-forum-cn#941] [General]
10.7日，专业版无故跳成免费版，**专业版到期日期：2027.1.26**，反馈至后台、邮箱无法收到恢复码，麻烦尽快处理一下！
![LT收据](https://github.com/user-attachments/assets/5a92193a-a2c2-4793-a4ee-e293c1db40fa)

**Comment by hzch170** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@hzch170 您好. 已经帮您提交问题上去了

**Reply by LuLu120325** [reply]
请帮我处理退款，给个明确的回复，谢谢，邮箱：2647611833@qq.com

**Reply by hzch170** [reply]
@Mike777Khoo 你好，帮忙看看进度，有问题抓紧时间解决！

**Reply by MaxMKMax** [reply]
@hzch170 您好, 这是您的专属激活码:
KMD8G-2DPTB-4CP7Y-8GWKH-PTPRK

**Comment by MaxMKMax** [comment]
@hzch170 您好. 请问您的问题解决了吗? 
谢谢您支持蓝灯!

**Reply by hzch170** [reply]
节点不稳定。另外原到期日期是2027.1.26，麻烦帮忙处理一下，谢谢！

**Reply by hzch170** [reply]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@hzch170 好的. 我再帮您提交问题.

**Comment by MaxMKMax** [comment]
@hzch170 

"关于节点的问题"

您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
